dipy/workflows/viz.py

```diff
@@ -0,0 +1,82 @@
+from os.path import join as pjoin
+from dipy.workflows.workflow import Workflow
+from dipy.io.streamline import load_tractogram
+from dipy.io.image import load_nifti
+# TODO: add support for peaks
+# from dipy.io.peaks import load_peaks
+from dipy.viz.app import horizon
+
+
+class HorizonFlow(Workflow):
+
+    @classmethod
+    def get_short_name(cls):
+        return 'horizon'
+
+    def run(self, input_files, cluster=False, cluster_thr=15.,
+            random_colors=False, length_lt=1000, length_gt=0,
+            clusters_lt=10**8, clusters_gt=0, native_coords=False,
+            stealth=False, out_dir='', out_stealth_png='tmp.png'):
+
+        """ Highly interactive visualization - invert the Horizon!
+
+        Parameters
+        ----------
+        input_files : variable string
+        cluster : bool
+        cluster_thr : float
+        random_colors : bool
+        length_lt : float
+        length_gt : float
+        clusters_lt : int
+        clusters_gt : int
+        native_coords : bool
+        stealth : bool
+        out_dir : string
+        out_stealth_png : string
+        """
+        verbose = True
+        tractograms = []
+        images = []
+        interactive = not stealth
+        world_coords = not native_coords
+
+        io_it = self.get_io_iterator()
+
+        for input_output in io_it:
+
+            f = input_output[0]
```

Comment:
Best to avoid single letter variable names, because they are hard to search for.